### DISCUSSION TITLE
Shortcuts in OS tray Menu and Expandable Menu in List View

### DISCUSSION THREAD

**Discussion by abstractindia** [pieces-app/support#80] [Ideas]
Suggesting some features for future.
(1) Add a shortcut "Open Pieces Desktop" in the Pieces OS tray right click menu.
(2) An expandable menu (Accordion Menu) in list view in Pieces Desktop based on language. Please refer to the image.
![Image_2024-02-08_10-06-55-M](https://github.com/pieces-app/support/assets/14857472/9a39d1c2-f2d0-4990-b027-3dff518791a6)

**Comment by eumel1968** [comment]
I agree with you!
